src/logic/schedule-exporter/schedule-export.logic.ts

```diff
@@ -161,4 +166,32 @@ export class ScheduleExportLogic {
       fs.saveAs(blob, filename);
     });
   }
+
+  private shiftInfoLogics(
```

Comment:
Szczerze mówiąc widziałem to nieco inaczej: myślałem że użyjesz bezpośrednio `ShiftHelper` w tym celu. 
Ale w sumie zgadzam się, że twoje podejście jest sensowniejsze, bo używamy tej samej logiki w obu miejscach. 
 Więc myślę, że na razie tak może być, bo w następnym sprincie będziemy robić refaktoring i jakoś to przekminimy.

Reply:
Szczerze powiedziawszy to myślałem, że po to właśnie zrobiłeś takie api w shift info logic żeby tego użyć xD

Reply:
Tak, ale chciałem używać go tylko w komponentach. 
Ale tej metody w tym miejscu też ma sens